modules/features/manatime_feature_timesheets/manatime_feature_timesheets.module

```diff
@@ -7,6 +8,29 @@
 include_once 'manatime_feature_timesheets.features.inc';
 
 /**
+ * Implements hook_menu().
+ */
+function manatime_feature_timesheets_menu() {
+  $items = array();
+
+  $items['timesheet/add-new-form/%/%ctools_js'] = array(
+    'page callback' => 'manatime_feature_timesheets_add_row_callback',
+    'page arguments' => array(2, 3),
+    'access callback' => 'user_is_logged_in',
+    'type' => MENU_CALLBACK,
+  );
+
+  $items['timesheet/remove-row-form/%/%/%ctools_js'] = array(
```

Comment:
Same here, for the argument that is the node, please change % to %node for efficiency.